routes/api_v1.js

```diff
@@ -108,6 +110,30 @@ route.get('/expand/:shortcode', function (req, res) {
 
 });
 
+route.post('/longURL/expand', function (req, res) {
```

Comment:
would prefer this to be `/search` instead of `/longURL/expand`
I am merging this. please make a new PR to fix this.